rasa/nlu/utils/pattern_utils.py

```diff
@@ -108,8 +120,9 @@ def extract_patterns(
     use_lookup_tables: bool = True,
     use_regexes: bool = True,
     use_only_entities: bool = False,
+    use_word_boundaries: bool = True,
 ) -> List[Dict[Text, Text]]:
-    """Extract a list of patterns from the training data.
+    r"""Extract a list of patterns from the training data.
```

Comment:
```suggestion
    """Extract a list of patterns from the training data.
```

Reply:
We need add a prefix `r` here, because without it, `make lint` will report `D301 Use r""" if any backslashes in a docstring`

Reply:
ok, wasn't aware of that